src/ert/_c_wrappers/enkf/config/field_config.py

```diff
@@ -66,12 +71,15 @@ def load(self, run_path: Path, real_nr: int, ensemble: EnsembleAccessor):
         trans = self.input_transformation
         data_transformed = field_transform(data, trans) if trans else data
         ensemble.save_field(key, real_nr, data_transformed)
+        _logger.debug(f"load() time_used {time.perf_counter() - t}s")
```

Comment:
Should we reduce the resolution a bit here?
Perhaps something like:

```python
_logger.debug(f"load() time_used {(time.perf_counter() - t):.4f}s")
```

Reply:
Yeas - any more digits does not make any sense